### ISSUE TITLE
Load neuron-region connectivity for all connectomes FAFB/L1EM/FlyWire/Optic lobe

### ISSUE THREAD

**Issue by dosumis** [VirtualFlyBrain/VFB_connectomics_import#8]
Regions are JFRC2 L&R.  But we don't have painted domains for these on JFRC2 - or visible on JRC2018U

Proposal: Robbie creates these domains **on JRC2018U** - as new individuals.  Where these correspond to 2018 painted domain the two should be linked.  Alex can then curate as for hemibrain.

**Comment by admclachlan** [comment]
Can now get synapse counts (weights) per region from CATMAID, not yet T-bars.

Need new individual JRC2018U rois to allow curation load.

**Comment by admclachlan** [comment]
- Find names in FAFB.
 - Dataset 'JenettShinomya_BrainName', make left/right individuals through curation interface. Type does not include left/right.
 - Make images?
 - TODO make separate ticket.

**Comment by dosumis** [comment]
Blocked by: https://github.com/VirtualFlyBrain/VFB_connectomics_import/issues/10

**Comment by admclachlan** [comment]
Expanding this to cover all connectomes
